crates/router/src/connector/globalpay/response.rs

```diff
@@ -29,7 +31,7 @@ pub struct GlobalpayPaymentsResponse {
     /// The country in ISO-3166-1(alpha-2 code) format.
     pub country: Option<String>,
     /// The currency of the amount in ISO-4217(alpha-3)
-    pub currency: Option<String>,
+    pub currency: Option<Currency>,
```

Comment:
why are we expecting Enum here, this might lead to deserialization error

Reply:
The currency of the amount in ISO-4217(alpha-3) so we wont get that error

Reply:
Still, i will prefer it to be a string. You can consult about this from anyone from the connector team. 
cc: @SamraatBansal

Reply:
It will work, as the enum is also used in the request, so request-response currency can never mismatch